actor-testkit-typed/src/main/scala/org/apache/pekko/actor/testkit/typed/internal/StubbedActorContext.scala

```diff
@@ -55,7 +55,7 @@ private[pekko] final class FunctionRef[-T](override val path: ActorPath, send: (
   override def provider: ActorRefProvider =
     throw new UnsupportedOperationException(
       "ActorRefs created for synchronous testing cannot be used as targets for asking. Use asynchronous testing instead. " +
-      "See https://doc.akka.io/docs/akka/current/typed/testing.html#asynchronous-testing")
+      "See https://pekko.apache.org/docs/akka/current/typed/testing.html#asynchronous-testing")
```

Comment:
there will be no akka in pekko.apache.org paths (ie the `docs/akka` bit) - Pekko team still needs to discuss what paths will end up looking like

Reply:
@Seetaramayya I published https://pekko.apache.org/ - if you change the URLs to use `docs/pekko` instead of `docs/akka` then they should work